apps/functions-subscription/api/internal.yaml

```diff
@@ -0,0 +1,159 @@
+openapi: 3.0.1
```

Comment:
**thought (non-blocking)**: should we use the latest OpenAPI version?

I see there is a version [`3.1.0` (which is the latest)](https://github.com/OAI/OpenAPI-Specification/releases/tag/3.1.0), and a [version `3.0.3`](https://github.com/OAI/OpenAPI-Specification/releases/tag/3.0.3) which is the latest related to the `3.0` version.

Did you use the version `3.0.1` due to some issues with the generator?

Reply:
Good catch! I kept the version provided on #16. Once merged, I can try if it works with the newer one as well, or we can try and bump it directly on #16.

Reply:
UPDATE: The version `3.1.0` raises the following error:
```
Error: SyntaxError: Unsupported OpenAPI version: 3.1.0. Swagger Parser only supports versions 3.0.0, 3.0.1, 3.0.2, 3.0.3
npm ERR! Lifecycle script `generate` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: functions-subscription@0.0.0 
npm ERR!   at location: /.../trial-system/apps/functions-subscription 
```
I'll bump it to `3.0.3`

Reply:
Done ✅